test/gurka/test_build_admin.cc

```diff
@@ -0,0 +1,168 @@
+#include <gtest/gtest.h>
+
+#include "gurka.h"
+
+#include "baldr/admin.h"
+#include "mjolnir/adminbuilder.h"
+#include "mjolnir/admin.h"
+#include "mjolnir/pbfgraphparser.h"
+
+using namespace valhalla;
+using namespace valhalla::baldr;
+using namespace valhalla::mjolnir;
+
+
+class AdminTest : public ::testing::Test
+{
+protected:
+  static gurka::map admin_map;
+
+  static void SetUpTestSuite() {
+    constexpr double gridsize = 10;
+
+    const std::string ascii_map = R"(
+          A-------B-------C
+          |       |       |
+          |       |       |
+          |       |       |
+          F-------E-------D
+    )";
+
+    // To define an administrative boundary, the nodes must for a closed polygon.
```

Comment:
s/must for a closed polygon/must **form** a closed polygon ?

Reply:
"Form".  Thanks.